01-path-basics/102-your-first-cluster/readme.adoc

```diff
@@ -100,7 +100,14 @@ To launch your worker nodes, run the following CloudFormation CLI command:
 
 The `AWS_STACK_NAME`, `EKS_WORKER_AMI`, `EKS_VPC_ID`, `EKS_SUBNET_IDS`, and `EKS_SECURITY_GROUPS` environment variables should have been set during the link:../101-start-here[Cloud9 Environment Setup].
 
-Node provisioning usually takes less than 5 minutes. You can query the status of your cluster with the following command. When your cluster status is `CREATE_COMPLETE`, you can proceed.
+Node provisioning usually takes less than 5 minutes.
+You may execute this command to wait until the create-stack has completed:
+
+    $ aws cloudformation wait stack-update-complete --stack-name k8s-workshop-worker-nodes
+
+Read more about `https://docs.aws.amazon.com/cli/latest/reference/cloudformation/wait/index.html[cloudformation wait, window="_blank"]`.
+
+You can also query the status of your cluster with the following command. When your cluster status is `CREATE_COMPLETE`, you can proceed.
```

Comment:
I'm not sure if these additional instructions add value to the content. Besides, it is not clear what the reader should expect after running the command

Reply:
Also, after I ran this command, my cli prompt just sat there even after CloudFormation finished creating the stack.

Reply:
Which command are you referring to? The one this line is on is a direct copy from the previous instructions.

Reply:
Hi @miketheman, I'm referring to this command
$ aws cloudformation wait stack-update-complete --stack-name k8s-workshop-worker-nodes